test/appium/tests/atomic/chats/test_public.py

```diff
@@ -328,6 +329,304 @@ def test_public_chat_open_using_deep_link(self):
+            self.chat_2.just_fyi("Sending Audio in chat")
+            self.chat_2.record_audio_message(message_length_in_seconds=3)
+            self.chat_2.send_message_button.click()
+
+        sending_list = {
+            "sticker": device_2_sends_sticker,
+            "image": device_2_sends_image,
+            "audio": device_2_sends_audio,
+        }
+
+        for key, value in sending_list.items():
+            navigate_to_start_state_of_both_devices()
+            sending_list[key]()
+            if not self.device_1.element_by_text_part(key.capitalize()).is_element_displayed(10):
+                self.errors.append("%s not appeared in Push Notification" % key.capitalize())
+                self.device_1.click_system_back_button()
+                self.device_1.get_app_from_background()
+            else:
+                self.device_1.element_by_text_part(key.capitalize()).click()
+            message = self.chat_2.chat_element_by_text(key)
+            self.chat_1.set_reaction(key)
+            if message.emojis_below_message(own=False) != 1:
+                self.errors.append("Counter of reaction is not set on %s for message receiver!" % key)
+            self.chat_1.set_reaction(key)
+            if message.emojis_below_message(own=False) == 1:
+                self.errors.append("Counter of reaction is not re-set on %s for message receiver!" % key)
+
+        self.chat_2.just_fyi("Sending Emoji/Tag/Links in chat")
+        # TODO: add link and tag messages after #11168 is fixed(rechecked 23.11.21, valid)
+        navigate_to_start_state_of_both_devices()
+
+        emoji_name = random.choice(list(emoji.EMOJI_UNICODE))
+        emoji_unicode = emoji.EMOJI_UNICODE[emoji_name]
+
+        self.chat_2.just_fyi("Sending Emoji in chat")
+        self.chat_2.chat_message_input.send_keys(emoji.emojize(emoji_name))
+        self.chat_2.send_message_button.click()
+
+        if not self.device_1.element_by_text_part(emoji_unicode).is_element_displayed(10):
+            self.errors.append("Emoji not appeared in Push Notification")
+            self.device_1.click_system_back_button()
+            self.device_1.get_app_from_background()
+        else:
+            self.device_1.element_by_text_part(emoji_unicode).click()
+
+        emoji_message = self.chat_2.chat_element_by_text(emoji_unicode)
+        self.chat_1.set_reaction(emoji_unicode, emoji_message=True)
+        if emoji_message.emojis_below_message(own=False) != 1:
+            self.errors.append("Counter of reaction is not set on Emoji for message receiver!")
+        self.chat_1.set_reaction(emoji_unicode, emoji_message=True)
+        if emoji_message.emojis_below_message(own=False) == 1:
+            self.errors.append("Counter of reaction is not re-set on Emoji for message receiver!")
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702067)
+    @marks.medium
+    def test_messaging_open_user_profile_long_press_on_message_1_1_chat(self):
```

Comment:
not sure it is worth separate case;
can be added as a check to any 1-1 chat test and removed.

Reply:
to say in truth, I would prefer to leave it as separate case than moving it as check to some other irrelevant 1-1 test. IMHO, adding this test as additional check in other test case goes against our goal of making more simple and atomic cases.  But if you insist, I can move this check to @marks.testrail_id(695843) test_edit_delete_message_in_one_to_one_. What do you think?

Reply:
decided to remove this check at all